sdk/purview/purview-workflow-rest/test/public/workflowtasks.spec.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// import { PurviewWorkflowClient } from "@azure-rest/purview-workflow";
+import { Recorder } from "@azure-tools/test-recorder";
+import { createClient } from "./utils/recordedClient";
+
+import { Context } from "mocha";
+import { PurviewWorkflowClient } from "../../src/clientDefinitions";
+import { ApproveApprovalTaskParameters, RejectApprovalTaskParameters } from "../../src/parameters";
+import { isUnexpected } from "../../src/isUnexpected";
+
+describe("Operate the workflow task.", () => {
+  let recorder: Recorder;
+  let client: PurviewWorkflowClient;
+  let workflowtaskId1: string;
+  let workflowtaskId2: string;
+
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    client = await createClient(recorder);
+    workflowtaskId1 = "721716fa-13b0-4613-beb5-87ffb5a3ce63";
+    workflowtaskId2 = "f0d9bf80-9490-40f9-8bc4-ea70aef701de";
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("should approve a workflow task.", async function () {
+    const options: ApproveApprovalTaskParameters = {
+      body: { comment: "Thanks for raising this!" },
+    };
+    const result1 = await client
+      .path("/workflowtasks/{taskId}/approve-approval", workflowtaskId1)
+      .post(options);
+
+    if (isUnexpected(result1)) {
+      throw result1.body.error;
+    }
```

Comment:
Agreed with Jose, could you add some assertion for response body?

Reply:
Thanks for your suggestion, for workflowtasks.spec.ts, there have no result in response, it just return status code, so I didn't verify result in this test. I added assertion for other tests.